### DISCUSSION TITLE
Welcome to concore-editor Discussions!

### DISCUSSION THREAD

**Discussion by pradeeban** [ControlCore-Project/concore-editor#60] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Rahuljagwani** [comment]
Hello everyone, I am Rahul Jagwani pursuing B.E.(CSE)  from DSCE Bangalore, India. I am currently in pre-final year. I have made few contributions in concore-editor repository and looking forward to complete this project in upcoming GSOC. 
It looks like there is no proper documentation available for the repository, only just enough information is present. There is no information available on each and every feature present in concore-editor. I will be more than happy to write the complete documentation regarding the same as I am getting familiar to the concore-editor with each and every contribution.
I request organization mentors if organization can participate in Google Season Of Docs(GSOD) so that proper documentation projects can also be done.

Thank You

**Reply by pradeeban** [reply]
Hi and welcome, @Rahuljagwani!

Good question. In fact, we were checking GSoD just yesterday but decided not to apply at this time for a few reasons.

* From the past years, it appears GSoD targets more established open-source organizations, unlike GSoC. There are no universities/departments/research labs, unlike GSoC.

* We could participate as CONTROL-CORE. But that may be too small of a project at this point.

* Too much admin overhead as a research department. For GSoD, Google will fund us (the organization) with the whole amount.
We should distribute the funding to the technical writers. This appears too complex for us, considering International money transfers will be involved.


concore itself has more documentation in the research paper - https://doi.org/10.1109/ACCESS.2022.3161471

The concore editor was developed as part of 2 GSoC. DHGWorkflow in GSoC 2021 and concore editor as a fork of DHGWorkflow in GSoC 2022. Both projects focused more on coding and did not give adequate time to documentation. I would recommend this project give a higher priority to documentation as well.

**Reply by Rahuljagwani** [reply]
Sure, understood what you are conveying. I want to ask one more thing if there are any other prerequisites I need to understand in order to start this project. Any feedbacks would be appreciated.

**Reply by pradeeban** [reply]
You are doing perfectly. One point to remember is - please keep an eye on the project descriptions page (https://github.com/NISYSLAB/Emory-BMI-GSoC/blob/main/README.md) for more updates. As you may have noticed, we keep improving/expanding on the ideas and the overall details based on discussions with potential collaborators like you.

**Reply by Rahuljagwani** [reply]
Definitely, I will be keeping an eye on that.
